cf_xarray/accessor.py

```diff
@@ -1732,7 +1732,8 @@ def guess_coord_axis(self, verbose: bool = False) -> DataArray | Dataset:
         """
         obj = self._obj.copy(deep=True)
         for var in obj.coords.variables:
-            if obj[var].ndim == 1 and _is_datetime_like(obj[var]):
+            var_is_coord = var in sum(obj.cf.coordinates.values(), [])
```

Comment:
```suggestion
            var_is_coord = any(var in val for val in obj.cf.coordinates.values())
```

is this easier to read?

Reply:
yes, let's go for that one